default.nix

```diff
@@ -87,12 +91,25 @@ stdenv.mkDerivation rec {
       with builtins; filterSource
         (path: _: !elem (baseNameOf path) [".git" "result" "bin"]) ./.;
 
+  enableParallelBuilding = true;
+
   prefixKey = "-prefix ";
 
   buildFlags = [ "world" ] ++ optional buildDoc "doc-html";
 
-  installTargets = [ "install" ] ++ optional buildDoc "install-doc-html";
+  postBuild = "make byte"; # Needed by the test-suite
+  # But it is not recommended to do it in parallel with make world
+
+  installTargets =
+    [ "install" "install-byte" ] ++ optional buildDoc "install-doc-html";
+
+  inherit doInstallCheck;
+
+  preInstallCheck = ''
+    patchShebangs tools/
+    patchShebangs test-suite/
+  '';
 
-  inherit doCheck;
+  installCheckTarget = [ "check" ];
```

Comment:
This includes `validate`, is that intended?
I don't think you can avoid setting the variables that the gitlab test suite job does to make it work with installed Coq, that's why I thought about having a separate installcheck target.

Reply:
Yes, it does include `validate`. I could restrict to `test-suite` if deemed sufficient.